import-export-cli/cmd/k8s/addApi.go

```diff
@@ -149,12 +146,12 @@ func createAPI(configMapName, timestamp string) {
 	k8sSaveConfig := true
 	if timestamp != "" {
 		//set update timestamp
-		apiCrd.Spec.UpdateTimeStamp = timestamp
+		apiCrd.Spec.UpdateTimeStamp = strings.Split(timestamp, "-")[1]
+		//fmt.Println(apiCrd.Spec.UpdateTimeStamp)
```

Comment:
Shall we remove this commented line?

Reply:
Fixed with aa03cd8b4d47fd7fec7dc79a709f5a9f8cba9fd1